soccer/src/soccer/soccer_mom/soccer_mom.hpp

```diff
@@ -0,0 +1,17 @@
+#include "rclcpp/rclcpp.hpp"
+#include "std_msgs/msg/string.hpp"
+#include <rj_msgs/msg/team_color.hpp>
+#include <rj_constants/topic_names.hpp>
+
+namespace tutorial {
+
+class SoccerMom : public rclcpp::Node {
+public:
+    SoccerMom();
+private:
```

Comment:
Make sure to adhere to the C++ docstrings requirements that are shown in the Contributions page of our docs. In future projects, it will be a requirement. For now, it is ok.